OWNERS

```diff
@@ -2,11 +2,17 @@
 approvers:
   - ecordell
   - njhale
-  - jpeeler
-  - dinhxuanvu 
+  - dinhxuanvu
```

Comment:
I wonder if we should just add everyone to `approvers` -- @ecordell thoughts?

Reply:
My methodology behind this is to pick 4 top code contributors as approvers. I thought it would be beneficial to cap number of approvers so not every PR gets approved right away and one lgtm will allow the PR to get merged too quickly without at least one more person has a chance to review and lgtm it. Just a guarding measure. However, if we want to add everyone, I'm fine with it.